operators/sumologic-kubernetes-collection-helm-operator/4.9.0-1/manifests/operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,556 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: "[\n    {\n        \"apiVersion\": \"helm-operator.sumologic.com/v1alpha1\"\
+      ,\n        \"kind\": \"SumologicCollection\",\n        \"metadata\": {\n   \
+      \         \"name\": \"collection\"\n        },\n        \"spec\": {\n      \
+      \      \"sumologic\": {\n                \"accessId\": \"\",\n             \
+      \   \"accessKey\": \"\",\n                \"clusterName\": \"kubernetes\",\n\
+      \                \"scc\": {\n                    \"create\": true\n        \
+      \        }\n            },\n            \"kube-prometheus-stack\": {\n     \
+      \           \"prometheus-node-exporter\": {\n                    \"service\"\
+      : {\n                        \"port\": 9200,\n                        \"targetPort\"\
+      : 9200\n                    }\n                }\n            },\n         \
+      \   \"otellogs\": {\n                \"daemonset\": {\n                    \"\
+      containers\": {\n                        \"otelcol\": {\n                  \
+      \          \"securityContext\": {\n                                \"privileged\"\
+      : true\n                            }\n                        }\n         \
+      \           },\n                    \"initContainers\": {\n                \
+      \        \"changeowner\": {\n                            \"securityContext\"\
+      : {\n                                \"privileged\": true\n                \
+      \            }\n                        }\n                    }\n         \
+      \       }\n            }\n        }\n    }\n]"
+    capabilities: Basic Install
+    categories: Logging & Tracing,Monitoring
+    containerImage: registry.connect.redhat.com/sumologic/sumologic-kubernetes-collection-helm-operator@sha256:f5bda9fe35c32141b1af41785e8b2ddfe386bd4ceef6209bdeb67d60d8becbaf
+    createdAt: '2024-07-19T10:48:00Z'
+    operators.operatorframework.io/builder: operator-sdk-v1.6.1+git
+    operators.operatorframework.io/project_layout: helm.sdk.operatorframework.io/v1
+    repository: https://github.com/SumoLogic/sumologic-kubernetes-collection-helm-operator.git
+    support: Sumo Logic
+    operators.openshift.io/valid-subscription: 'Apache License v2'
+    features.operators.openshift.io/disconnected: true
+    features.operators.openshift.io/fips-compliant: false
+    features.operators.openshift.io/proxy-aware: true
+    features.operators.openshift.io/tls-profiles: false
+    features.operators.openshift.io/token-auth-aws: false
+    features.operators.openshift.io/token-auth-azure: false
+    features.operators.openshift.io/token-auth-gcp: false
+    features.operators.openshift.io/cnf: false
+    features.operators.openshift.io/csi: false
```

Comment:
Looks like the `feature` label values should be in `""`:
    features.operators.openshift.io/fips-compliant: "false"
    features.operators.openshift.io/proxy-aware: "true"
    features.operators.openshift.io/tls-profiles: "false"
    features.operators.openshift.io/token-auth-aws: "false"
    features.operators.openshift.io/token-auth-azure: "false"
    features.operators.openshift.io/token-auth-gcp: "false"
    features.operators.openshift.io/cnf: "false"
    features.operators.openshift.io/csi: "false"